nixtlats/data/tsloader.py

```diff
@@ -12,7 +12,7 @@
 from fastcore.foundation import patch
 from torch.utils.data import DataLoader
 
-from .tsdataset import TimeSeriesDataset
+from .tsdataset import TimeSeriesDataset, WindowsDataset
 
 # Cell
 class TimeSeriesLoader(DataLoader):
```

Comment:
The type of the dastaset should be now `Union[TimeSeriesDataset, WindowsDataset]`.